### DISCUSSION TITLE
What should be the results type for Cypress Mocha reporter

### DISCUSSION THREAD

**Discussion by nischaytv** [test-results-reporter/reporter#36] [Q&A] [answered]
Hi All,

I'm using mocha framework

What should be the report type for junit or what are the different types available
![image](https://user-images.githubusercontent.com/20622111/173549915-b94c0564-2c1e-4cc3-9526-6b9f5e34900b.png)

**Comment by ASaiAnudeep** [comment, marked as answer]
Hello @nischaytv, 

With mocha, we can use the default **json** format or **junit** format by using additional reporters.

The type could be either
- **mocha** - [default reporter](https://mochajs.org/#json)
- **junit** - [mocha-junit-reporter](https://www.npmjs.com/package/mocha-junit-reporter)

See [docs](https://test-results-reporter.github.io/guides/results.html#arguments) for more info.

###### Thanks for reaching out. If you find the tool useful, please support by giving us a star.
